packages/logging-system/src/SetLocalStorage.ts

```diff
@@ -1,36 +1,32 @@
 import { LogRequestList, LogResponse, LogType } from './types/LogType';
 
 export const SetLocalStorageClear = () => {
-  const list: any[] = [];
+  const list: LogType[] = [];
   localStorage.setItem('yls-web', JSON.stringify(list));
 };
 
 export const SetLocalStorage = async (
   logger: LogType,
   putLog: (data: LogRequestList) => Promise<LogResponse>
 ) => {
-  if (window.localStorage.getItem('yls-web') == undefined) {
-    const list: any[] = [];
-    list.push(logger);
-    localStorage.setItem('yls-web', JSON.stringify(list));
-  } else {
-    const remainList: any[] = JSON.parse(localStorage.getItem('yls-web') as string) || [];
-    if (remainList.length < 10) {
-      const updateList = [...remainList, logger];
-      localStorage.setItem('yls-web', JSON.stringify(updateList));
-    } else {
-      const req: LogRequestList = {
-        logRequestList: remainList,
-      };
+  const list: LogType[] = JSON.parse(localStorage.getItem('yls-web') as string) || [];
```

Comment:
```suggestion
  const list: LogType[] = JSON.parse(localStorage.getItem('yls-web') as string) ?? [];
```
`||` 말고 [nullish coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing) 연산자(`??`) 활용하시면 의도에 맞으실 것 같습니다.